tests/test_io/test_land_surface_vertices.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import pandas as pd
+from subsurface.io.grids import land_surface_vertices
+import os
+
+input_path = os.path.dirname(__file__)+'/../data'
+
+
+@pytest.fixture(scope="module")
+def get_vertices() -> pd.DataFrame:
+    fp = input_path + "/land_surface_vertices.csv"
+    vertices = land_surface_vertices.read_in_land_surface_vertices(fp)
+    return vertices
+
+
+def test_land_surface_vertices_nrows(get_vertices):
```

Comment:
Do you know why this is 413250? The csv is 132696. At the moment this two test are the only one failing

Reply:
I think I didn't use the tests for a long time as I generalized the script fit for all csv files as surface_reader and when I last used the tests the csv still had more rows until I cut them down to only 132695

Reply:
Maybe we can simply delete them then. You test suite is already awesome

Reply:
Okay and actually we don't need land_surface_reader anymore, do we? All functionalities are covered by surface_reader as well now